routes/feeds.js

```diff
@@ -49,6 +49,7 @@ router.get('/feed-chat/:feedId', isAuth, feed.getOneFeedChatService);
 router.post('/open-graph', isAuth, feed.getOpenGraph);
 router.delete('/:postId', isAuth, feed.deletePost);
 router.delete('/anonymous/:postId', isAuth, feed.deleteAnonymousPost);
+router.delete('/anonymous-v2/:postId', isAuthAnonim, feed.deleteAnonymousPostV2);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/anonymous-v2/:postId` uses a different authentication middleware `isAuthAnonim`. If this is intended to authenticate anonymous users, please ensure that it properly checks for valid tokens or credentials. Also, consider renaming `isAuthAnonim` to something more descriptive and in line with the existing naming conventions, such as `isAuthAnonymous`.
```diff
- router.delete('/anonymous-v2/:postId', isAuthAnonim, feed.deleteAnonymousPostV2);
+ router.delete('/anonymous-v2/:postId', isAuthAnonymous, feed.deleteAnonymousPostV2);
```
If `isAuthAnonim` is not intended to authenticate anonymous users, you might want to use the existing `isAuth` middleware instead.
```diff
- router.delete('/anonymous-v2/:postId', isAuthAnonim, feed.deleteAnonymousPostV2);
+ router.delete('/anonymous-v2/:postId', isAuth, feed.deleteAnonymousPostV2);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->